src/style.css

```diff
@@ -1 +1,61 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.body {
+  margin: 0;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  margin: 20px 8px 0;
+}
+
+.form-field {
+  display: none;
+}
+
+.search {
+  font-weight: 300;
+  background-image: url(images/Icon.png);
+  background-repeat: no-repeat;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  outline: none;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
```

Comment:
You should always provide a fallback font in case the main one doesn't work. So, if the Avenir font fails to load for some reason, the browser will use the sans-serif font. This is a good practice to ensure that your website has a consistent look across different browsers and platforms. You can specify multiple fallback fonts by separating each font name with a comma. The browser will use the first font in the list that is installed on the computer or that can be downloaded using a @font-face at-rule.